src/jsonrpc/mod.rs

```diff
@@ -42,6 +42,7 @@ pub struct JsonRpcClientImpl {
 
 impl JsonRpcClientImpl {
     /// Creates a client that sends all requests to `url`.
+    #[allow(dead_code)]
```

Comment:
This `dead_code` directive should not be needed once you rebase from master, right? The `LotusClient` makes use of the `JsonRpcClientImpl` IIRC, right?

Reply:
let me try